src/components/RegistrationList.tsx

```diff
@@ -123,6 +125,7 @@ export const RegistrationListItemContent = ({
             getTitleString(entityDescription?.mainTitle)
           ) : (
             <MuiLink
+              target={target ?? ''}
```

Comment:
Det var ønsker om at de skulle se mulig duplikat i et annet vindu.

Reply:
Er ikke enig i ønsket, da det skal være brukerens valg om man ønsker å åpne lenker i ny- eller samme fane, mens man her tvinges til å åpne i ny fane.
Vanlig å åpne eksterne lenker til andre domener i ny fane, mens interne lenker overlates til bruker, litt for at de ikke skal slippe helt ut av sfæren da 😅 
Anyways: skal ikke legge meg mer borti om det åpnes i ny fane eller ikke 😇